include/experimental/__p0009_bits/extents.hpp

```diff
@@ -521,10 +526,13 @@ template <class IndexType, size_t... Extents> class extents {
   MDSPAN_INLINE_FUNCTION friend constexpr bool
   operator==(const extents &lhs,
              const extents<OtherIndexType, OtherExtents...> &rhs) noexcept {
-    bool value = true;
-    for (size_type r = 0; r < m_rank; r++)
-      value &= rhs.extent(r) == lhs.extent(r);
-    return value;
+    if constexpr (rank() != extents<OtherIndexType, OtherExtents...>::rank()) {
+      return false;
+    } else {
+      for (size_type r = 0; r < m_rank; r++)
+        if(rhs.extent(r) != lhs.extent(r)) return false;
```

Comment:
```suggestion
      for (size_type r = 0; r < m_rank; r++) {
        if (rhs.extent(r) != lhs.extent(r)) {
          return false;
        }
      }
```

Reply:
Proposing to run the whole thing through clang-format and starting to enforce it.